### PR TITLE
Events forwarder deployment

### PR DESCRIPTION
**Pull request by AneethAnand**

### CODE REVIEW

File: sysdigcloud/events-forwarder-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: events-forwarder
+    spec:
+      containers:
+        - name: events-forwarder
+          image: quay.io/sysdig/events-forwarder:8d1a093-05-21-2019-01-03-53
```
Comment (mbreitung) [inline sysdigcloud/events-forwarder-deployment.yaml]: this version looks like an internal build number. Can this be converted to semantic versioning?

Comment (AneethAnand) [reply sysdigcloud/events-forwarder-deployment.yaml]: @mbreitung  Can you please give me an example where Semantic versioning is used?

Comment (mbreitung) [reply sysdigcloud/events-forwarder-deployment.yaml]: We are converting to semantic versioning in the next release for all components. Previously it was simply the build number which is much cleaner than a hash code+date

Comment (AneethAnand) [reply sysdigcloud/events-forwarder-deployment.yaml]: @mbreitung , so can you merge this change then?

Comment (mbreitung) [reply sysdigcloud/events-forwarder-deployment.yaml]: This needs a proper version

File: sysdigcloud/events-forwarder-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
```
Comment (mbreitung) [inline sysdigcloud/events-forwarder-deployment.yaml]: please add `---` to the top of this file

**Comment by mbreitung** [comment]
Closing this stale PR

**Pull request closed without merging**